include/cantera/thermo/ThermoPhase.h

```diff
@@ -1172,7 +1185,179 @@ class ThermoPhase : public Phase
 
     //@}
 
+    //! @name Set Mixture Composition by Mixture Fraction
+    //! @{
+
+    //! Set the mixture composition according to the
+    //! mixture fraction = kg fuel / (kg oxidizer + kg fuel)
+    /*!
+     * Fuel and oxidizer compositions are given either as
+     * mole fractions or mass fractions (specified by `basis`)
+     * and do not need to be normalized. Pressure is kept constant.
+     * Elements C, S, H and O are considered for the oxidation.
+     *
+     * @param mixFrac    mixture fraction (between 0 and 1)
+     * @param fuelComp   composition of the fuel
+     * @param oxComp     composition of the oxidizer 
```

Comment:
There are trailing spaces on this line and a few others. Can you clean these up? They're pretty easy to spot if you use `git log --patch --color`.
```suggestion
     * @param oxComp     composition of the oxidizer
```